packages/app/src/cli/services/deploy/upload.test.ts

```diff
@@ -88,67 +87,24 @@ describe('uploadFunctionExtensions', () => {
       extension.outputPath = joinPath(tmpDir, 'index.wasm')
       await writeFile(extension.outputPath, '')
       const uploadURLResponse: UploadUrlGenerateMutationSchema = {
-        data: {
-          uploadUrlGenerate: {
-            headers: {},
-            maxSize: '200',
-            url: uploadUrl,
-            moduleId: 'module-id',
-          },
+        uploadUrlGenerate: {
+          headers: {},
+          maxSize: '200 kb',
```

Comment:
I guess the error message used to be "less than 200" and the test just mocked it? 😆